src/gnn_tracking/models/graph_construction.py

```diff
@@ -25,6 +26,34 @@
 from gnn_tracking.utils.log import logger
 from gnn_tracking.utils.torch_utils import freeze_if
 
+def ground_truth_classifier(data: Data, mode: bool=False):
```

Comment:
Functions where the signature depends on a parameter should be avoided IMO, I'll just make a `TruthNoiseClassifierModel` and remove this function